### DISCUSSION TITLE
Examples for EON tutorial

### DISCUSSION THREAD

**Discussion by AlanSong641** [TheochemUI/eOn#121] [General]
Hi everyone, here I updated the examples used for the eon tutorial. 👍 

[CECAM_examples.tar.gz](https://github.com/user-attachments/files/15972367/CECAM_examples.tar.gz)

**Comment by HaoZeke** [comment]
@AlanSong641 could you add more of the examples here?

**Comment by AlanSong641** [comment]
hi, here is the example with dynamics method which is not working yet.
[pt-heptamer-neb.zip](https://github.com/user-attachments/files/15987265/pt-heptamer-neb.zip)
